inc/dashboard-widgets/namespace.php

```diff
@@ -10,6 +10,9 @@
  * Bootstrap.
  */
 function bootstrap() {
+	if ( function_exists( 'classicpress_version' ) ) {
```

Comment:
`if ( function_exists( 'wp_ajax_get-community-events' ) ) { `

It’s great to check for classic press conflicts, but if we can make our checks universal (in case someone else makes a fork and wants to use this) then it’s better 🙂